File: Outputs/QR-Scanner/Java/final_QR Scanner.js.java
import android.widget.TextView; 
import android.view.View; 
import android.widget.Button; 
import android.widget.EditText; 
import android.widget.ImageView; 
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
 import androidx.core.app.ActivityCompat; 
public class QR Scanner.js extends AppCompatActivity { 
@Override
            protected void onCreate (Bundle savedInstanceState){
              super.onCreate(savedInstanceState);
              setContentView(R.layout.activity_main);

              myqrcodebarcodescanner qr = new myqrcodebarcodescanner();
// <StatusBarstyle="auto"/>
// <Viewstyle={styles.container}><RNCameraref={ref=>{this.camera=ref;}}style={{flex:1,width:'100%',}}onGoogleVisionBarcodesDetected={this.barcodeRecognized}></RNCamera><StatusBarstyle="auto"/></View>
} 
 }

File: Outputs/QR-Scanner/Swift/final_QR Scanner.js.swift
import CodeScanner; 
import SwiftUI 
struct ContentView:View{ 
@State var value:Bool = false 
 var body:some View {
                        VStack{

        
          @State private var showingScanner = false
            Button(action: {
                self.showingScanner = true
            }) {
                Image(systemName: "qrcode")
                    .resizable()
                    .frame(width: 32, height: 32)
            }
            .sheet(isPresented: $showingScanner) {
                CodeScannerView(codeTypes: [.qr], simulatedData: "-") { result in
                     switch result {
                     case .success(let code):
                         print(code)
                     case .failure(let error):
                         print(error)
                     }
                     self.showingScanner = false
                 }
            }
            // <StatusBarstyle="auto"/>
// <Viewstyle={styles.container}><RNCameraref={ref=>{this.camera=ref;}}style={{flex:1,width:'100%',}}onGoogleVisionBarcodesDetected={this.barcodeRecognized}></RNCamera><StatusBarstyle="auto"/></View>
} 
 }
 } 
struct ContentView_Previews:PreviewProvider{ 
static var previews:some View{ 
ContentView() } 
 }